sdk/purview/purview-account-rest/src/index.ts

```diff
@@ -8,4 +8,6 @@ export * from "./models";
 export * from "./parameters";
 export * from "./responses";
 
+export { paginate, PaginateReturn, GetArrayType } from "./paginateHelper";
```

Comment:
Just out of curious why we don't use `export * from "./paginateHelper";` the same as other files.